--- sw-fe-container/src/app/login/login.component.spec.ts
+++ sw-fe-container/src/app/login/login.component.spec.ts
@@ -48,7 +48,7 @@
         { provide: AuthService, useValue: mockAuthService },
         { provide: Router, useValue: mockRouter },
         { provide: DevAuthApiService, useValue: mockDevAuthApi },
-        { provide: APP_CONFIG, useValue: { apiPath: "/api", logLevel: "DEBUG" } },
+        { provide: APP_CONFIG, useValue: { apiPath: "/api", logLevel: "DEBUG", auth: false } },
         LoggerService,
       ],
     }).compileComponents();
